akka-docs/rst/general/stream/stages-overview.rst

```diff
@@ -1,21 +1,373 @@
 .. _stages-overview:
 
-###############################################
 Overview of built-in stages and their semantics
-###############################################
+===============================================
 
-All stages by default backpressure if the computation they encapsulate is not fast enough to keep up with the rate of
+
+Source stages
+-------------
+These built-in sources are available from `akka.stream.scaladsl.Source` and `akka.stream.javadsl.Source`
+
+
+
+fromIterator
+^^^^^^^^^^^^
+Streams the values from an iterator, requesting the next value when there is demand. If the iterator
+performs blocking operations, make sure to run it on a separate dispatcher.
+
+*emits* when the next value returned from the iterator
+*completes* when the iterator reaches it's end
+
+single
+^^^^^^
+Streams a single object
+
+*emits* the value once
+*completes* when the single value has been emitted
+
+repeat
+^^^^^^
+Streams a single object repeatedly
+
+*emits* the same value repeatedly when there is demand
+*completes* never
```

Comment:
these do not render vertically but treated as the same paragraph. Or does Github render differently than the Sphinx output?